scripts/test-run.sh

```diff
@@ -12,16 +12,14 @@
 # ------------------------------------------------------------------------------
 
 # ------------------------------------------------------------------------------
-# Environment Variables
+# Configuration
 # ------------------------------------------------------------------------------
 
+RELEASE_NAME="${RELEASE_NAME:-chart-tests}"
+RELEASE_NAMESPACE="${RELEASE_NAMESPACE:-$(uuidgen)}"
 TEST_ENV_NAME="${TEST_ENV_NAME:-kong-charts-tests}"
 TAG="${TAG:-default}"
 
-# ------------------------------------------------------------------------------
-# Shell Configuration
-# ------------------------------------------------------------------------------
-
 set -euo pipefail
```

Comment:
ditto - this needs to happen before any other command invocations (`uuidgen` in this case)

Reply:
:+1: eeecd15f23143944bb98ee162b8ec4cf53a6f6e6